mozci/push.py

```diff
@@ -1190,10 +1190,13 @@ def classify_regressions(
         else:
             all_groups = push_groups
 
+        groups_confirmed = {g.name: g.is_confirmed_failure() for g in all_groups}
```

Comment:
note: I had an issue finding tasks when using this below similar to `g.is_consistent_failure(...)`, so I went this route where I got the status for each group name and then verified that name.

Reply:
Do you remember what the issue was? In theory it should just work the same way :/

Reply:
the issue was not all groups were shown, specifically groups from a -cf task.  I cannot remember what specifically what the root cause; I think there was a filter of some kind taking place earlier.